src/rootReducer.js

```diff
@@ -0,0 +1,18 @@
+const initState = {

```

Comment:
You can create a type for this. It can be an interface or type that has a key count and a type for its value.